packages/module/src/RemoveModal/RemoveModal.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react';
+import { Button, Checkbox, Modal, ModalProps, ModalVariant, Split, SplitItem, Stack, TextContent } from '@patternfly/react-core';
+import { createUseStyles } from 'react-jss'
+
+
+const useStyles = createUseStyles({
+  rbacDeleteIcon: {
+    marginRight: 'var(--pf-v5-global--spacer--xs)'
+  },
+})
```

Comment:
this should not be needed
```suggestion
const useStyles = createUseStyles({
  rbacDeleteIcon: {
    marginRight: 'var(--pf-v5-global--spacer--xs)'
  },
})
```